client/components/domains/domain-search-results/style.scss

```diff
@@ -23,3 +23,22 @@
 		}
 	}
 }
+
+.domain-search-results__transfer-card {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	font-size: 17px;
+}
+
+.domain-search-results__transfer-card-copy {
+	margin-left: 10px;
```

Comment:
There is no left margin here on the comp. (Though you may want a right margin to ensure space between the copy and the link at small screen widths.)